src/components/pages/Gameification/GameModal.js

```diff
@@ -0,0 +1,47 @@
+//** Import Modules */
+import React, { useEffect } from 'react';
+import { gsap } from 'gsap';
+
+export default function GameModal(props) {
+  // Get the data to display
+  const { title, description, buttonTxt } = props.modalData;
+
+  // Functio to close the modal
+  const closeModal = () => {
+    gsap.to('.modal-container', { y: '100vh', duration: 0.5 });
+    gsap.to('#notification-modal', { opacity: 0, duration: 0.5, delay: 0.5 });
+    gsap.to('#notification-modal', {
+      display: 'none',
+      duration: 0.1,
+      delay: 1,
+    });
+
+    setTimeout(() => {
+      props.disableModalWindow();
+    }, 1000);
```

Comment:
Try to see if the value 1000 can be moved to a constant variable.

Reply:
Ok, got it

Reply:
Alrighty. done